site/docs/try/configuration.md

```diff
@@ -73,11 +77,54 @@ When setting `nessie.version.store.type=DYNAMO` which enables DynamoDB as the ve
 | `quarkus.dynamodb.sync-client.type`     | `url`          | `url, apache` | Sets the type of the sync HTTP client implementation                                                                                                |
 
 !!! info
-    A complete set of DynamoDB configuration options for Quarkus can be found on [quarkiverse.github.io](https://quarkiverse.github.io/quarkiverse-docs/quarkus-amazon-services/dev/amazon-dynamodb.html#_configuration_reference)
+A complete set of DynamoDB configuration options for Quarkus can be found on [quarkiverse.github.io](https://quarkiverse.github.io/quarkiverse-docs/quarkus-amazon-services/dev/amazon-dynamodb.html#_configuration_reference)
+
+#### MongoDB Version Store Settings
+
+When setting `nessie.version.store.type=MONGODB` which enables MongoDB as the version store used by the Nessie server, the following configurations are applicable in combination with `nessie.version.store.type`:
+
+| Property                            | Default values | Type     | Description                     |
+|-------------------------------------|----------------|----------|---------------------------------|
+| `quarkus.mongodb.database`          |                | `String` | Sets MongoDB database name.     |
+| `quarkus.mongodb.connection-string` |                | `String` | Sets MongoDB connection string. |
+
+!!! info
+A complete set of MongoDB configuration options for Quarkus can be found on [quarkus.io](https://quarkus.io/guides/all-config#quarkus-mongodb-client_quarkus-mongodb-client-mongodb-client)
 
 ### Version Store Advanced Settings
 
-The following configurations are advanced configurations to configure how Nessie will store the data into the configured data store:
+The following configurations are advanced configurations are only needed for **legacy** version stores to configure how Nessie will store the data into the configured data store:
```

Comment:
If these are needed only for legacy version store, it should be part of the table at line 127?